doc/howto/storage_pools.md

```diff
@@ -218,89 +218,6 @@ To show detailed information about a specific pool, use the following command:
 (storage-resize-pool)=
 ## Resize a storage pool
 
-If you need more storage, you can increase the size of your storage pool.
+If you need more storage, you can increase the size of your storage pool by changing the `size` configuration key:
```

Comment:
We should mention here this only works for loop file backed storage pools where the loop file is managed by LXD.